app/templates/admin/users/users.html

```diff
@@ -60,6 +51,59 @@
             </a>
         </div>
     </div>
+    <div class="display-feature-table">
```

Comment:
A minor issue: the radio buttons move up and down as you shrink the screen. This happened even with the old table layout. Should I fix it or leave it?
![screen shot 2015-04-24 at 5 10 24 pm](https://cloud.githubusercontent.com/assets/7966567/7328622/e759f440-eaa4-11e4-92ab-26e470d3b6ca.png)
![screen shot 2015-04-24 at 5 10 16 pm](https://cloud.githubusercontent.com/assets/7966567/7328623/e776efd2-eaa4-11e4-90a7-2fef30ef4c30.png)